diff --git a/vendor/github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1/typo.go b/vendor/github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1/typo.go
new file mode 100644
index 00000000..56d10239
--- /dev/null
+++ b/vendor/github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1/typo.go
@@ -0,0 +1,3 @@
+package v1
+
+var SchemaGroupVersion = SchemeGroupVersion
